tests/Unit/FunctionsTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace
+{
+    require __DIR__ . '/../../bridge/functions.php';
+}
+
+namespace DDTrace\Tests\Unit
+{
+    final class FunctionsTest extends BaseTestCase
+    {
+        protected function setUp()
+        {
+            parent::setUp();
+            putenv('DD_TRACE_ENABLED');
+            putenv('DD_DISABLE_URI');
+        }
+
+        public function testTracerEnabledByDefault()
+        {
+            $this->assertTrue(\DDTrace\Bridge\dd_tracing_enabled());
+        }
+
+        public function testTracerDisabled()
+        {
+            putenv('DD_TRACE_ENABLED=false');
+            $this->assertFalse(\DDTrace\Bridge\dd_tracing_enabled());
+        }
+
+        public function testTracerEnabledForUrisByDefault()
+        {
+            $this->assertTrue(\DDTrace\Bridge\dd_tracing_route_enabled());
+        }
+
+        /**
+         * @dataProvider urisDataProvider
+         * @param string $uri
+         * @param bool $expected
+         */
+        public function testTracerEnabledOrDisabledForUris($uri, $expected)
+        {
+            putenv('DD_DISABLE_URI=/foo,/users/*,/bar/*/test,/index.php?foo=*');
+
+            $this->assertSame(
+                $expected,
+                \DDTrace\Bridge\dd_tracing_route_enabled($uri)
+            );
+        }
+
+        public function urisDataProvider()
+        {
+            return [
+                ['', true],
+                ['/', true],
+                ['/foo', false],
+                ['/users', true],
+                ['/users/' . mt_rand(), false],
+                ['/bar/test', true],
+                ['/bar/' . mt_rand() . '/more/test', false],
```

Comment:
I was originally thinking that `*` was greedy, but I can change it to, "matches everything except slashes," if that's more standard/expected. :)

Reply:
This is a good point. What I am concerned about is that if we include `/` in `*` there will be edge cases that we are not able to cover. While if we exclude `/` from `*` it may be a little bit more cases to write, but they will be able to cover (almost) every use case.

Again I am almost nit-picking here, so feel free to apply your best judgement 😄